src/Microsoft.Dnx.Watcher.Core/DnxWatcher.cs

```diff
@@ -182,41 +181,37 @@ private void AddProjectAndDependeciesToWatcher(IProject project, IFileWatcher fi
 
             foreach (var projFile in project.ProjectDependencies)
             {
-                //var fullProjectFilePath = Path.Combine(Path.GetDirectoryName(project.ProjectFile), projFile);
```

Comment:
I think that was a bug in my code, it should actually do the commented line because projects in the lock file are relative to the project's path. If the project being watched references other projects, it might not watch correctly the dependencies of those projects